.vscode/cspell.dictionaries/jargon.wordlist.txt

```diff
@@ -54,6 +55,7 @@ hashsums
 infile
 iflag
 iflags
+impls
```

Comment:
Alternatively, ignore the spelling error on the line where it's imported (if that's supported) and rename it to `implementations` or another word that doesn't get flagged for subsequent uses.

Reply:
I think this is fine, impl (and hence impls) are such common words in Rust-related writing that it belongs on the jargon list. Might be better suited to put under abbreviations though?

Reply:
Good call, done!

Reply:
Oh interesting hahaha, I was looking at a different abbreviations list (in `jargon.wordlist.txt`) but this is fine! Thanks!